index.js

```diff
@@ -0,0 +1,71 @@
+const MOVE_UP = 'up'
+const MOVE_DOWN = 'down'
+const SELECT = 'select'
+
+const initialState = {
+  items: [
+    '1',
+    '2',
+    '3',
+    '4',
+    '5',
+    '6',
+    '7',
+    '8',
+    '9',
+    '10',
+  ],
+
+  selected: -1,
+}
+
+console.log("initialState", initialState.items);
+
+const action = (operation, element=null) => {
+  
+  return {
+    type: operation,
+    select: element,
+  }
+}
+
+const reducer = (state, action) => {
+
+  const interimState = [...state.items];
+
+    switch (action.type) {
+
+      case MOVE_DOWN:
+        
+        if(state.selected !== -1) {
+          interimState
+            .splice(state.selected-1, 2, state.items[state.selected], state.items[state.selected-1])
+        }
```

Comment:
remove these empty lines
```suggestion
    switch (action.type) {
      case MOVE_DOWN:
        if(state.selected !== -1) {
          interimState
            .splice(state.selected-1, 2, state.items[state.selected], state.items[state.selected-1])
        }
```